src/Pages/Main/SortBtnFilter.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+import 'antd/dist/antd.css';
+
+const SortBtnFilter = ({ sortBtn, sortValue }) => {
+  return (
+    <div>
+      <TypeInbox>
+        <Type>종류</Type>
+        <SortInbox>
+          <Button onClick={e => sortBtn(e)} value={0}>
```

Comment:
- event 는 자동으로 인자에 들어갑니다. 따라서 현재와 같이 별도의 함수를 (`e => sortBtn(e)`) 만드실 필요 없이, 다음과 같이 작성해주세요!
```suggestion
          <Button onClick={sortBtn} value={0}>
```

Reply:
수정했습니다!